superset/assets/src/dashboard/components/FilterIndicator.jsx

```diff
@@ -56,12 +62,14 @@ class FilterIndicator extends React.PureComponent {
     return (
       <FilterTooltipWrapper tooltip={filterTooltip}>
         <div
-          className="filter-indicator"
+          className={`filter-indicator ${
+            isFilterFieldActive ? 'isFilterFieldActive' : ''
```

Comment:
this css class naming seems inconsistent, is there anything wrong with just adding on the class `active` along with `filter-indicator`? Or at least using dash case to match `filter-indicator`

Reply:
now use `active`.